README.md

```diff
@@ -1,2 +1,79 @@
 # pupil-icon-font
 Icon font for Pupil software
+
+Using fontello toolchains to locally optimize and convert individual SVGs to TTF font
+
+- 
+- Uses `svg-font-create` and
+
+## Installation
+
+You need [node.js])(https://nodejs.org/en/download/) and `fontforge` installed first.
+
+### Debian Ubunbu
+Fontforge
+```bash
+sudo add-apt-repository ppa:fontforge/fontforge;
+sudo apt-get update
+sudo apt-get install fontforge;
+```
+
+Dependencies
+```
+npm install
+```
+
+## Make .ttf font
+
+Run command to make ttf font
+
+```
+make
+```
+
+## Project structure
+
+Diagram of repo hierarchy
+
+```
+/pupil-icon-font
+|-- svgs
+	`-- individual svgs
+|-- font
+	`-- glyph svg and ttf file
+|-- dist
+	`-- temp file for transformed svg
+```
+
+## Development	
+
+Create fork and clone your repo locally.
+
+### New SVG icons 
+Add icons to `./svgs` folder,
+Edit config.yml and add your icon name and unicode there.
```

Comment:
Edit `config.yml` to specify your icon names and unicode mapping for each glyph.